Makefile

```diff
@@ -613,13 +624,28 @@ $(LINUXKIT): | $(GOBUILDER)
 	git clone $(LINUXKIT_SOURCE) /tmp/linuxkit && \
 	cd /tmp/linuxkit && \
 	git checkout $(LINUXKIT_VERSION) && \
+	if [ -e /eve/$(LINUXKIT_PATCHES_DIR) ]; then \
+	    patch -p1 < /eve/$(LINUXKIT_PATCHES_DIR)/*.patch; \
+	fi && \
 	cd /tmp/linuxkit/src/cmd/linuxkit && \
 	GO111MODULE=on CGO_ENABLED=0 go build -o /go/bin/linuxkit -mod=vendor . && \
 	cd && \
 	rm -rf /tmp/linuxkit" \
 	$(GOTREE) $(GOMODULE) $(BUILDTOOLS_BIN)
 	$(QUIET): $@: Succeeded
 
+$(YQ): | $(GOBUILDER)
+	$(QUIET)$(DOCKER_GO) \
+	"unset GOFLAGS; rm -rf /tmp/linuxkit && \
+	git clone $(YQ_SOURCE) /tmp/yq && \
+	cd /tmp/yq && \
+	git checkout $(YQ_VERSION) && \
+        go build -o /go/bin/yq . && \
```

Comment:
Please define `$(YQ): GOOS=$(shell uname -s | tr '[A-Z]' '[a-z]')` and use `CGO_ENABLED=0` to be compatible with MacOS.